.github/ISSUE_TEMPLATE/core_bug.md

```diff
@@ -0,0 +1,91 @@
+---
+name: Core Bug
+about: "Report a bug you have encountered using SeAT. Are considered as bugs everything including ACL outage, security outage, Exception, etc... **Note:** Issues related to SeAT API must be reported on the API Issue template. If you're encountering issues while attempting to install SeAT itself, please go to our Slack in #support channel which will be a better please to sort such problems : [Get an invite](https://eveseat-slack.herokuapp.com)"
```

Comment:
`sort such problems : ` 
Should be:
`sort such problems: ` (there is no space between word and punctuation marks)